client/App.jsx

```diff
@@ -34,22 +34,22 @@ class App extends React.Component {
       }
     })
       .then((result) => {
-        //console.log(result.data[0], '--------------')
         this.setState({
           game: result.data[0]
         })
       })
   }
 
   render() {
-    console.log(this.state.game.summary, '++++++++++++++++++')
     return (
       <div>
-        <h3>All Games > Action Games > GAME.TITLE</h3>
-        <h1>GAME.TITLE</h1>
+        <h3 className='filePath'> <a href=''>All Games</a> > <a href=''>Action Games</a> > <a href=''>{this.state.game.summary.title}</a></h3>
+        <h1 className='title'>{this.state.game.summary.title}</h1>
```

Comment:
I think it would be better for readability if you first store the game you are working with in a shorter variable. e.g. const {game} = this.state;
Then you would say. const {title} = game.summary;
On line 47, you'd then say: {title}